example/src/components/Participants.tsx

```diff
@@ -75,7 +82,7 @@ export const Participants = ({ participants, onPress }: ParticipantsProp) => {
               key={p.participant.id + p.trackId}
               style={[
                 getStylesForParticipants(),
-                isParticipantSpeaking(p)
+                isParticipantSpeaking(p) && isVideoTrackScreenshare(p) === false
```

Comment:
it would be better to include check for screensharing to `isParticipantSpeaking`

Reply:
Done.